scripts/validate_container.sh

```diff
@@ -54,7 +70,7 @@ echo "Testing backends:"
 echo "$requested_backends"
 echo
 
-if $missing_backend; 
+if $missing_backend || [ "$available_backends" == "" ]; 
```

Comment:
`[ -z "$available_backends" ]`

Do we need two variables here? It seems like if `$available_backends` has 0 length, then the backend is missing. Or if the user selects a backend that isn't in the list of available backends.

Reply:
I'll copy over the edit, but I am not sure what you are suggesting regarding the two variables (meaning missing_backend and available_backends). I want to print what backends were detected as being available. I am missing how to compute if a user requested a backend that is not available without creating the missing_backend variable.